### ISSUE TITLE
Python3

### ISSUE THREAD

**Issue by edsu** [edsu/pymarc#149]
I think we can drop python2 support now that is no longer being maintained. This amounts to removing our use of the six module, and pymarc will have no dependencies afterwards!

**Comment by Wooble** [comment]
Probably a good time to stop supporting EOL'ed py3 versions, too. 3.3 and 3.4 are both unsupported. dropping 3.5 too would be nice since f-strings are so nice, but it's technically still eligible for security bugfixes until September.

**Comment by herrboyer** [comment]
Dropping 3.5 will also let us use proper typing instead of using comments :

```python
    # py3.5
    def foo(self, bar):
        # type: (str) -> str
        return bar

    # py3.6
    def foo(self, bar: str) -> str:
        return bar
```

**Comment by termim** [comment]
IMHO it is good time for this. Please consider PR #152 as a first step.

**Comment by edsu** [comment]
Fixed by #152!
